src/content/blog/posts/iceberg-athena-dbt.md

```diff
@@ -0,0 +1,116 @@
+
+#### Orchestration
+
+We use [GitHub Actions](https://github.com/features/actions) to automate and orchestrate the dbt builds. Co-locating the SQL and automation code enhances transparency and simplifies maintenance. We also added several features to facilitate early error detection and automated recovery:
+
+* A development GitHub workflow which is triggered when a pull request is raised. This workflow selectively builds new or updated models and their children in a development environment, making it easier for users to review the transformation prior to deploying to production.
+
+* Integrating Slack notifications into our production workflows to promptly alert stakeholders in the event of model or validation failures.
+
+* Incorporating a retry mechanism into our production workflows with custom logic. This feature identifies and automatically reattempts failed models and their children, which is particularly useful for resolving errors stemming from transient issues.
+
+### Bringing it all together
+
+We previously utilized Athena alongside Glue PySpark for our ELT pipelines. Migrating from Glue PySpark and Hive to Athena v3, Iceberg, and dbt has resulted in an impressive 99% reduction in individual query costs. This cost efficiency has enabled us to switch from weekly to more frequent daily refreshes while also onboarding new pipelines. Notably, we have still managed to achieve substantial savings, as shown in our monthly service cost graph:
+
+![](https://raw.githubusercontent.com/moj-analytical-services/dmet-cfe/main/investigations/iceberg_athena_dbt/images/glue_vs_athena_costs.png)
+
+In addition, the runtime for our longest jobs has decreased by 75%, and intermittent failures due to insufficient resources have become extremely rare. During the migration, we took the opportunity to enhance our dbt solution by integrating features that improve both maintainability and data quality. This includes dynamically generating models and implementing a Write-Audit-Publish (WAP) pattern. These improvements ensure that our analysts have more timely access to large datasets, and to work more efficiently on a reliable and maintainable platform.
+
+Furthermore, the unification of our data processing tools fosters a culture of collaboration within our data teams, making it easier to share enhancements and best practices. Looking ahead, we are excited about the potential to further innovate and refine our analytics capabilities, ensuring we continue to deliver greater value to the justice system.
+
+### Acknowledgements
+
+Bla bla bla
```

Comment:
Before this gets published - less `bla bla bla`!

Reply:
don't worry I'll fix as part of the next PR :-)

What did you think of the flow? I'm a bit worried because I added the glue comparison a bit further down the line which throws off the narrative